jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/genericfilter/GenericFilter.java

```diff
@@ -373,6 +378,27 @@ public void setAutoApply(boolean autoApply) {
         }
     }
 
+    /**
+     * @return {@link KeyCombination} that is used to apply the filter
+     */
+    @Nullable
+    public KeyCombination getApplyShortcut() {
+        return applyButton.getShortcutCombination();
+    }
+
+    /**
+     * Sets a new {@link KeyCombination} to apply the filter.
+     *
+     * @param applyShortcut string representation of a {@link KeyCombination}
+     */
+    public void setApplyShortcut(String applyShortcut) {
```

Comment:
It seems that method can take Nullable parameter. Check it and if so, add annotation.